<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(51, 51, 51)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="btnwidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="btnlayout"/>
   </widget>
   <widget class="QWidget" name="main_costumewidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>500</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="main_costumelayout"/>
   </widget>
   <widget class="QWidget" name="sub_costumewidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="sub_costumelayout"/>
   </widget>
   <widget class="QWidget" name="deformwidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>370</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="deformlayout"/>
   </widget>
   <widget class="MyGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <zorder>openGLWidget</zorder>
   <zorder>btnwidget</zorder>
   <zorder>main_costumewidget</zorder>
   <zorder>sub_costumewidget</zorder>
   <zorder>deformwidget</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>myglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
